### PR TITLE
Feat design improvements

### PR DESCRIPTION
**Pull request by fuenkchen**
* Implements this task: https://3.basecamp.com/3500782/buckets/1333491/todos/850052136 and fixes https://github.com/nzzdev/Q-election-executive/issues/40.
* If no party is given the line is left blank except for `others` group.
* With these changes some of the styling could be simplified/made clearer, so maybe also watch out for obsolete "leftovers" in the code
* prettier in a separate commit
* deployed on staging

### CODE REVIEW

File: styles_src/default.scss [hunk 1]
```diff
@@ -56,16 +54,26 @@ $image-margin: 7px;
...
+      height: 100%;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      width: calc(100% - (#{$image-size} + #{$image-margin}));
+    }
+
     &-bar {
+      width: 100%;
       float: left;
```
Comment (benib) [inline styles_src/default.scss]: removing this `float: left;` would make the padding-top value for the majority group more clear as the bar would not float out of context for the elements appearing in the DOM later. Also: this is probably a leftover from a previous design? At least I do not see why it should float, but I could very well miss something.

File: styles_src/default.scss [hunk 2]
```diff
@@ -116,15 +130,19 @@ $image-margin: 7px;
...
         white-space: nowrap;
         text-overflow: ellipsis;
-        padding-left: 5px;
+        margin-top: 2px;
       }
 
       &-votes {
         margin-left: auto;
-        z-index: 1;
+        /*         z-index: 1;
```
Comment (manuelroth) [inline styles_src/default.scss]: Can the z-index be removed here?

File: styles_src/default.scss (hunk 1 above)
Comment (fuenkchen) [reply styles_src/default.scss]: yes indeed, and I was wondering why I have to add the margin on majority line and arrow. Now I know and will remove these stuff.

File: styles_src/default.scss (hunk 2 above)
Comment (fuenkchen) [reply styles_src/default.scss]: yes, this is also a leftover of the pseudo transparency.

File: styles_src/default.scss (hunk 1 above)
Comment (fuenkchen) [reply styles_src/default.scss]: very nice catch, that way I could remove the height of each candidate item as well which also had bad side effects on the margin

**Pull request merged**